addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -252,29 +252,29 @@ img.ms-auto, img.mx-auto {
 
 @for $index from 1 through 4 {
     .display-#{$index}-fs {
-        font-size: map-get($display-font-sizes, $index);
+        @include font-size(map-get($display-font-sizes, $index));
     }
 }
 .h1-fs {
-    font-size: $h1-font-size;
+    @include font-size($h1-font-size);
 }
 .h2-fs {
-    font-size: $h2-font-size;
+    @include font-size($h2-font-size);
 }
 .h3-fs {
-    font-size: $h3-font-size;
+    @include font-size($h3-font-size);
 }
 .h4-fs {
-    font-size: $h4-font-size;
+    @include font-size($h4-font-size);
 }
 .h5-fs {
-    font-size: $h5-font-size;
+    @include font-size($h5-font-size);
 }
 .h6-fs {
-    font-size: $h6-font-size;
+    @include font-size($h6-font-size);
 }
 .base-fs {
-    font-size: $font-size-base;
+    @include font-size($font-size-base);
 }
```

Comment:
Not sure this is usefull as RFS starts when fs is bigger than 20px. Same for small font size...
Let me know what do you think @qsm-odoo 

![image](https://github.com/odoo/odoo/assets/78849981/1061b70c-3b49-44e7-86e7-c45196b95b5c)

Reply:
I would always use the mixin, whatever the font-size value is. It's more consistent but you also never know how the default behavior of that function can be tweaked in custo.